### DISCUSSION TITLE
Unwanted text next to logo

### DISCUSSION THREAD

**Discussion by DMunch69** [fastfetch-cli/fastfetch#982] [Q&A] [answered]
Hello, i have a problem that when i use fastfetch it prints the line "in]" next to my logo as shown below.
![image](https://github.com/fastfetch-cli/fastfetch/assets/131600543/bac0871e-4497-478f-b6bd-5522b5471159)
Is there a way to fix this? I have looked in the fastfetch config and there is no "in]".

**Comment by CarterLi** [comment, marked as answer]
Try making your terminal wider

**Reply by DMunch69** [reply]
I don't know if i should feel good that it wasn't an actual problem or bad that i spent so long trying to find the problem.
